src/components/ErrorModal.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div
+    class="modal-mask"
```

Comment:
```suggestion
    class="error-modal"
```
proposing to name the root node the same as the component